blueprints/ember-electron/index.js

```diff
@@ -24,96 +21,79 @@ class EmberElectronBlueprint extends Blueprint {
   afterInstall(/* options */) {
     let logger = new Logger(this);
 
-    return this._installDependencies(logger)
-      .then(() => this._createResourcesDirectories())
-      .then(() => this._ensurePackageJsonConfiguration())
-      .then(() => this._updateGitignore())
```

Comment:
Don't we still need to update `.gitignore` to include `electron-out`? Or does forge take care of that for us? In any case, in #189 we'll need to add `electron-livereload`.